src/api/constants/index.ts

```diff
@@ -143,6 +143,11 @@ export const ACCOUNT = {
   MAX_AUTH_RETRIES_TIMEFRAME: DATE_24_HOURS_IN_THE_PAST(),
 };
 
+export const DEFAULT_ENCRYPTION_SAVE_OBJECT = {
+  value: '',
```

Comment:
Any reason we are saving constants as an empty string.
Perhaps could do with some documentation please.